config/routes.rb

```diff
@@ -90,7 +90,7 @@
     member do
       post :restore
     end
-    resources :form, controller: "case_contacts/form"
+    resources :form, controller: "case_contacts/form", only: %i[show update]
```

Comment:
these are the only actions in the controller, so this is not a behavioral change (except preventing controller errors)